playbooks/thoth-openshift-trigger-build/run.yaml

```diff
@@ -12,3 +12,19 @@
         follow_redirects: all
         headers:
           Content-Type: "application/json"
+      when: webhook_url is defined
```

Comment:
Can we get some error message if none of booth tasks worried out?!

Reply:
Sure! Just added a debug msg display for the error message when none of the jobs is triggered.